scipy/spatial/_geometric_slerp.py

```diff
@@ -0,0 +1,206 @@
+from __future__ import division, print_function, absolute_import
+
+__all__ = ['geometric_slerp']
+
+import math
+import numpy as np
+from . import _slerp
+from scipy.spatial.distance import euclidean
+
+def geometric_slerp(start,
+                    end,
+                    t,
+                    tol=1e-7):
+    """
+    Spherical linear interpolation along unit-radius
+    great circle arc in arbitrary dimensional space.
+
+    Parameters
+    ----------
+    start : (D, ) array-like
+        Single D-dimensional input coordinate in a 1-D array-like
+        object.
+        Starting coordinate for interpolation will be converted
+        to float64.
+    end : (D, ) array-like
+        Single D-dimensional input coordinate in a 1-D array-like
+        object.
+        End coordinate for interpolation will be converted to
+        float64.
+    t: array
+        An array of doubles representing interpolation parameters,
+        with values required in the inclusive interval between
+        0 and 1. A common approach is to generate the array with
+        ``np.linspace(0, 1, n_pts)`` for linearly spaced points.
+        Ascending, descending, and scrambled orders are permitted.
+    tol: float
+        The absolute tolerance for determining if the start and end
+        coordinates are antipodes. Antipode detection will
+        raise an exception because of the ambiguity in the path.
```

Comment:
> Antipode detection will raise an exception because of the ambiguity in the path

Do you want to move this to a `Raises` section?  Just a suggestion.

Reply:
Ok, I've done as you requested here & transplanted to a `Raises` section.